package.json

```diff
@@ -15,11 +15,21 @@
     "registry": "https://registry.npmjs.org"
   },
   "devDependencies": {
+    "babel-core": "^6.7.4",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-react": "^6.5.0",
     "css-loader": "^0.23.1",
     "jsx-loader": "^0.13.2",
     "less": "^2.6.1",
-    "less-loader": "^2.2.2",
+    "less-loader": "^2.2.3",
     "react": "^0.14.7",
-    "style-loader": "^0.13.0"
+    "react-dom": "^0.14.7",
+    "style-loader": "^0.13.0",
+    "webpack": "^1.12.14",
+    "webpack-dev-server": "^1.14.1"
+  },
+  "scripts": {
+    "dev-server": "node_modules/.bin/webpack-dev-server --content-base docs/ --inline --watch"
```

Comment:
`webpack-dev-server --inline` adds a bit of js into the bundle that detects changes and reloads the page for you. So far I've found it a lot better than `webpack --watch` + opening `index.html` + reloading manually.